internal/wrapnix/wrapper.sh.tmpl

```diff
@@ -29,4 +29,9 @@ export {{ .ShellEnvHashKey }}_GUARD=true
 eval "$(DO_NOT_TRACK=1 devbox shellenv -c {{ .ProjectDir }})"
 fi
 
+# So that we do not invoke other bin-wrappers from
+# this bin-wrapper. Instead, we directly invoke the binary from the nix store, which
```

Comment:
nit, it's the binary of the profile (which I guess points to the nix store path)

Reply:
the emphasis here is on _binary_ as opposed to _wrapper_